src/net.cpp

```diff
@@ -55,6 +58,7 @@ using namespace std;
 
 namespace {
     const int MAX_OUTBOUND_CONNECTIONS = 8;
+    const int MAX_FEELER_CONNECTIONS = 1;
```

Comment:
This number seems to serve to increase MAX_OUTBOUND_CONNECTIONS as it is not used other than adding a number to this constant.